src/libraries/Directory.Build.props

```diff
@@ -103,6 +103,14 @@
     <IsPackable Condition="$(MSBuildProjectName.StartsWith('Microsoft.Extensions.')) and '$(IsSourceProject)' == 'true'">true</IsPackable>
   </PropertyGroup>
 
+  <Import Sdk="Microsoft.DotNet.PackageValidation" Project="Sdk.props" Condition="'$(IsPackable)' == 'true' and '$(IsSourceProject)' == 'true'"/>
+
+  <PropertyGroup Condition="'$(IsPackable)' == 'true'">
```

Comment:
Not sure we should be checking "IsPackable" in a props file.  Typically that's set in CSProj body.

Reply:
Yes, that should be set in Directory.Build.targets instead.